Signal/src/ViewControllers/CallViewController.swift

```diff
@@ -338,16 +338,30 @@ class CallViewController: OWSViewController, CallObserver, CallServiceObserver {
 //                                                action:#selector(didPressTextMessage))
         audioSourceButton = createButton(imageName:"audio-call-speaker-inactive",
                                           action:#selector(didPressAudioSource))
+        audioSourceButton.accessibilityLabel = NSLocalizedString("CALL_VIEW_AUDIO_SOURCE_LABEL",
+                                                                 comment: "Accessibility label for selection the audio source")
+        
         hangUpButton = createButton(imageName:"hangup-active-wide",
                                     action:#selector(didPressHangup))
+        hangUpButton.accessibilityLabel = NSLocalizedString("CALL_VIEW_HANGUP_LABEL",
+                                                            comment: "Accessibility label for hang up call")
+        
         audioModeMuteButton = createButton(imageName:"audio-call-mute-inactive",
                                            action:#selector(didPressMute))
+        audioModeMuteButton.accessibilityLabel = NSLocalizedString("CALL_VIEW_AUDIO_MUTE_LABEL",
+                                                                   comment: "Accessibility label for muting the microphone during audio call")
+        
         videoModeMuteButton = createButton(imageName:"video-mute-unselected",
                                            action:#selector(didPressMute))
+        videoModeMuteButton.accessibilityLabel = NSLocalizedString("CALL_VIEW_VIDEO_MUTE_LABEL", comment: "Accessibility label for muting the microphone during video call")
```

Comment:
Sorry - one last nit: 

Can you combine CALL_VIEW_VIDEO_MUTE_LABEL and CALL_VIEW_AUDIO_MUTE_LABEL into just "CALL_VIEW_MUTE_LABEL" to save some work for our translators. 

It's really unlikely that we'd ever want a different translation for the two buttons.

Reply:
Done :)